tools/distrib/python/grpcio_tools/grpc_tools/protoc.py

```diff
@@ -37,6 +38,9 @@ def main(command_arguments):
       command_arguments: a list of strings representing command line arguments to
           `protoc`.
     """
+    # command_arguments += [
```

Comment:
Still needed?

Reply:
No we don't need those anymore, deleted.